app/src/entry-point.lisp

```diff
@@ -170,7 +170,12 @@
      :type boolean
      :optional t
      :initial-value nil
-     :documentation "Disable printing of the wavefunction in batch mode.")))
+     :documentation "Disable printing of the wavefunction in batch mode.")
+
+    (("debugger" #\D)
```

Comment:
This is a duplicate of the `--debug` option (see line 143). Is that intentional? (Seems like the docstr should be changed at least.)

Reply:
Good catch!